dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -31,7 +31,8 @@ def user_ml_model_names
   # GET api/v1/ml_models/:model_id
   # Retrieve a trained ML model from S3
   def get_trained_model
-    model = download_from_s3(params[:model_id])
+    model_id = params[:model_id] ? params[:model_id] : UserMlModel.where(user_id: current_user.id).last.model_id
```

Comment:
Is it a long term plan for this API to return the last-trained model for a user if no ID is specified?